src/View/BaseViewTrait.php

```diff
@@ -75,14 +82,45 @@ private function getProps(): array
         return $props;
     }
 
+    /**
+     * Any viewVars that should not be included in Inertia `page`
+     * are set here
+     *
+     * @param mixed $props The view vars
+     * @param mixed $nonInertiaVars Array of non Inertia Vars
+     * @return array
+     */
+    private function setNonInertiaVars($props, $nonInertiaVars): array
```

Comment:
Is it used? I don't see it used anywhere so you can safely remove it.

Reply:
Will get rid of that